src/Helpers/JsonLd.php

```diff
@@ -59,8 +60,21 @@ public static function prepareDataForSerialization($obj)
             $data["type"] = self::getType($obj);
         }
 
-        // Add context only if a BaseModel (not RPDE stuff)
-        if(is_subclass_of($obj, "\\OpenActive\\BaseModel")) {
+        // Only add context if object is subclass of BaseModel
+        // and no parent, or parent is an RPDE item
+        if(
+            is_subclass_of($obj, "\\OpenActive\\BaseModel") &&
+            (
+                $parent === null ||
+                (
+                    // While instanceof will not throw an error
```

Comment:
personally would fine this a lot cleaner to read without the comment.

Reply:
Whoops was a bit too quick on merging :D - I've removed this on master